client/components/token-field/index.jsx

```diff
@@ -183,6 +183,12 @@ var TokenField = React.createClass( {
 			} );
 		}
 
+		if ( autoCapitalize ) {
+			Object.assign( props, {
+				autoCapitalize: autoCapitalize
+			} );
```

Comment:
Optional, but we can add a little [sugar](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Spread_operator) and do:
```javascript
props = { ...props, autoCapitalize };
```